### PR TITLE
Stats and Live payload formating

### PR DESCRIPTION
**Pull request by g-getsov**

### CODE REVIEW

File: stats_subevents.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{
+    "eventId": 2457,
+    "eventName": "English Premier League Matches",
+    "subeventId": 87010376,
+    "subeventName": "Everton v Man Utd",
+    "subeventType": "match",
+    "inRunning": true,
+    "startTime": "20161204T160000Z",
+	"statsInfo": {
+		"venueName": "White Hart Lane",
```
Comment (martofrog) [inline stats_subevents.json]: we can lose it, no needed for stats

File: stats_subevents.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		"participants": {
+			"homeTeam": {
+				"name": "Everton",
+				"shortName": "Everton",
+				"history": {
+					"matches": [
+						{
+							"result": "Lost",
```
Comment (martofrog) [inline stats_subevents.json:19]: do we have a date for it?

File: stats_subevents.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# /api/v1/stats/subevent/{subeventId}
+
+{
+    "eventId": 2457,
+    "eventName": "English Premier League Matches",
+    "subeventId": 87010376,
+    "subeventName": "Everton v Man Utd",
+    "subeventType": "match",
```
Comment (martofrog) [inline stats_subevents.json:8]: shall we also add the squads?

**Review by martofrog — CHANGES_REQUESTED** [review]

File: stats_subevents.json (hunk 2 above)
Comment (g-getsov) [reply stats_subevents.json:19]: We don't have the date unfortunately

File: stats_subevents.json (hunk 1 above)
Comment (g-getsov) [reply stats_subevents.json]: Agreed

File: stats_subevents.json (hunk 3 above)
Comment (g-getsov) [reply stats_subevents.json:8]: Yes we we shall

File: stats_subevents.json (hunk 2 above)
Comment (martofrog) [reply stats_subevents.json:19]: mmmh worth asking Kev, 'cause the will want to now the order recent to oldest

**Review by martofrog — APPROVED** [review]

**Pull request merged**